src/main/java/edu/cornell/kfs/coa/document/CUAccountMaintainableImpl.java

```diff
@@ -153,5 +184,65 @@ public List getSections(org.kuali.rice.kns.document.MaintenanceDocument document
         
         return super.getSections(document, oldMaintainable);
     }
+    
+    @Override
+    public void processAfterCopy(MaintenanceDocument document, Map<String, String[]> parameters) {
+    	document.getNotes().add(getNewBoNote(CUKFSConstants.AccountCreateAndUpdateNotePrefixes.ADD));
+    	super.processAfterCopy(document, parameters);
+    }
+    
+    @Override
+    public void processAfterNew(MaintenanceDocument document, Map<String, String[]> requestParameters) {
+    	document.getNotes().add(getNewBoNote(CUKFSConstants.AccountCreateAndUpdateNotePrefixes.ADD));
+    	super.processAfterNew(document, requestParameters);
+    }
+    
+    @Override
+	public void processAfterPost(MaintenanceDocument document, Map<String, String[]> requestParameters) {
```

Comment:
I added this override so that the text for the note that is automatically added when an account is edited is updated to include the document description. The doc description cannot be changed after doc is routed so this customized code is only called when doc is in status Initiated or Saved.